src/api/forms/service.js

```diff
@@ -176,7 +176,9 @@ export async function updateDraftFormDefinition(formId, definition, author) {
           {
             $set: {
               'draft.updatedAt': now,
-              'draft.updatedBy': author
+              'draft.updatedBy': author,
+              updatedAt: now,
+              updatedBy: author
```

Comment:
Thanks @KiranVarma02 for adding these

Do you mind updating the comment above? It only says we're updated the draft state

```js
// Update the `updatedAt/By` fields of the draft state
```